hue/hue/base/hue-mysql-dc.yaml

```diff
@@ -0,0 +1,114 @@
+---
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  name: hue-mysql
+  annotations:
+    template.alpha.openshift.io/wait-for-ready: "true"
```

Comment:
What is this doing?

Reply:
https://docs.openshift.com/container-platform/4.2/openshift_images/using-templates.html#templates-waiting-for-readiness_using-templates

I have no idea if it actually has any meaningful impact for ODH operator though. It comes from the initial implementation in the legacy ODH.